### PR TITLE
Separate contexts for fetching joind in data from raffle

### LINKED ISSUE #94: Separate contexts for fetching joind in data from raffle
We are now mixing domain contexts in context files, it was nice but it doesnt make sense to keep it as is.

In order to have properly separated contexts, based on domain & layer
For maintainers
We need to split Application and Api Contexts into relevant contexts
Whereas we had multiple domains in same context.

### PR DESCRIPTION
**Pull request by msvrtan**
In order to have properly separated contexts, based on domain & layer
For maintainers
We need to split Application and Api Contexts into relevant contexts
Whereas we had multiple domains in same context.
 
Closes #94

### CODE REVIEW

**Review by HcsOmot — APPROVED** [review]
👍

File: features/bootstrap/JoindInApiContext.php [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+class JoindInApiContext extends BaseContext
+{
+    use FixturesTrait;
+    /**
+     * @var KernelInterface
+     */
+    private $kernel;
+
+    /** @var string|null */
+    private $raffleId;
```
Comment (HcsOmot) [inline features/bootstrap/JoindInApiContext.php:16]: Not sure where this variable is used in this context? Does JoindInApiContext need to be aware of our raffler system?

File: features/bootstrap/JoindInApiContext.php [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    /** @var array|null */
+    private $picked;
```
Comment (HcsOmot) [inline features/bootstrap/JoindInApiContext.php:18]: Same here as previous^^

File: features/bootstrap/JoindInApiContext.php (hunk 1 above)
Comment (msvrtan) [reply features/bootstrap/JoindInApiContext.php:16]: We still need the ID, it's used in the URL

File: features/bootstrap/JoindInApiContext.php (hunk 2 above)
Comment (msvrtan) [reply features/bootstrap/JoindInApiContext.php:18]: Same answer, we need to store the information between steps

**Review by HcsOmot — COMMENTED** [review]
^^

**Pull request merged**